### PR TITLE
Name map element track sequence flags and use them

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/ride/station.c [hunk 1]
```diff
@@ -281,9 +281,11 @@ static void ride_invalidate_station_start(rct_ride *ride, sint32 stationIndex, s
     if (mapElement == NULL)
         return;
 
-    mapElement->properties.track.sequence &= 0x7F;
-    if (dl != 0)
-        mapElement->properties.track.sequence |= 0x80;
+    mapElement->properties.track.sequence &= ~MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT_MASK;
+    if (greenLight)
+    {
+        mapElement->properties.track.sequence |= MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT;
```
Comment (rwjuk) [inline src/openrct2/ride/station.c]: Shouldn't these be the other way around?
```
mapElement->properties.track.sequence &= ~MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT;
if (greenLight)
{
mapElement->properties.track.sequence |= MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT_MASK;
}
```

Comment (IntelOrca) [inline src/openrct2/ride/station.c]: You don't need a mask for a single bit. Just do:
```
mapElement->properties.track.sequence &= ~MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT;
if (greenLight)
{
mapElement->properties.track.sequence |= MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT;
}
```

Comment (Gymnasiast) [inline src/openrct2/ride/station.c]: @rwjuk No, it shouldn't. You use a mask to check, not to set.
@IntelOrca I'll change it.

Comment (IntelOrca) [inline src/openrct2/ride/station.c]: You can use a mask to clear as well.

**Pull request merged**